parquet-column/src/test/java/org/apache/parquet/column/values/bloomfilter/TestBlockSplitBloomFilter.java

```diff
@@ -113,6 +113,33 @@ public void testEquals() {
     assertNotEquals(bloomFilterTwo, bloomFilterThree);
   }
 
+  @Test
+  public void testUnion() {
```

Comment:
Can we add negative tests? For example, test checkCompatibility() return false

Reply:
Done!